app/controllers/todo_controller.rb

```diff
@@ -0,0 +1,9 @@
+class TodoController < ApplicationController
+
+  layout false
+  def index
+  end
+
+  def tasksOverview
```

Comment:
In ruby we don't use camel case style but the snake_case.
I recommend to start using Rubocop, very useful tool.

https://github.com/rubocop/rubocop